src/test/rustdoc-ui/generate-link-to-definition-opt2.stderr

```diff
@@ -0,0 +1,2 @@
+error: --generate-link-to-definition option can only be used with HTML output format
```

Comment:
Can you also test that the new flag requires -Zunstable-options?

Reply:
Done!